R/weighted_efficacy.R

```diff
@@ -0,0 +1,275 @@
+#' Assign N doses to age groups based on weightings by number of people eligible to be vaccinated
+#'
+#' @param doses Total available doses
+#' @param target_pop Number of people eligible to be vaccinated
+assign_doses <- function(doses, target_pop){
+  if(sum(target_pop) < doses){
```

Comment:
Should this be:
```suggestion
  if(sum(target_pop) <= doses){
```

This will also help prevent code fails that occur if doses = 0 and target_pop